### PR TITLE
Added dependency to documentation about testing with database

### LINKED ISSUE #7309: Testing with databases, 'jdbc' dependency missing in documentation
### API Scala

When I try to test my database code using - https://www.playframework.com/documentation/2.5.x/ScalaTestingWithDatabases it was not helpful because I can't even import `Databases` from `play.api.db.Databases`. I wasted a lot of time before I finally find out that I need to have.
```
libraryDependencies ++= Seq(
  jdbc
)
```
I used slick in my project and I don't have `jdbc` enabled in my dependencies. I think it should be matched in documentation, before any code. I am not sure could it be useful with slick ...

### PR DESCRIPTION
**Pull request by vooolll**
## Fixes

Fixes #7309 

## Purpose

To improve documentation about testing with databases

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @vooolll,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: documentation/manual/working/scalaGuide/main/tests/code/database/databases.sbt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//
+// Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com>
+//
+
+//#content
+libraryDependencies ++= Seq(
+  jdbc
```
Comment (wsargent) [inline documentation/manual/working/scalaGuide/main/tests/code/database/databases.sbt]: To test with a database backend, you only need:

```
libraryDependencies += jdbc % Test
```

Restricting the dependency to test scope means that it doesn't get added to the artifacts pushed to production.  If the project itself depends on a database, then adding it to compile is appropriate.

**Comment by vooolll** [comment]
@wsargent Good point, I've done changes to restrict dependencies to test scope.

**Comment by marcospereira** [comment]
Thank you, @vooolll. 

Do you mind also adding this to the Java documentation? It is the `JavaTestingWithDatabases.md` file. But instead of using `jdbc` you need to use `javaJdbc`.

**Comment by vooolll** [comment]
@marcospereira Sure, I have also updated java guide.

**Pull request merged**